solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -353,7 +383,10 @@ contract RandomBeacon is IRandomBeacon, Ownable {
         dkg.setSubmitterPrecedencePeriodLength(20); // TODO: Verify value
 
         relay.initSeedEntry();
-        relay.setRelayRequestFee(200e18);
+        // total gas of submit+approve ~780849. Assume gas price is 200gwei
+        // total dkg gas cost: 0.0000002 * 780849 = 0.1561698
+        // there need to be 5 requests to trigger dkg, so 0.156 / 5 = ~0.031 ETH need to be provided to break even.
+        relay.setRelayRequestFee(31e15); // TODO: revisit. Do we want to add a markup?
```

Comment:
> The parameters we took into account when calculating the relay request fee may change so the fee should be alterable

The relay request fee is changeable and can be updated by the governance in the current implementation. 

> On the other hand, we have problems with the contract size and adding one more public function makes it only worse. I would vote for eliminating the fee completely and just reverting in requestRelayEntry if !authorizedContracts[msg.sender]

Sure, I thought it's a must-have functionality which is described in the issue [here](https://github.com/keep-network/keep-core/issues/2824). But if we're okay removing it, then yes, it reduces the size.
We can certainly require only `authorizedContracts[msg.sender]`, but let's keep in mind that having a separate contract that will charge a fee (if needed) will increase the overall cost for the relay entry.

Request fee is removed here [f11d300](https://github.com/keep-network/keep-core/pull/2860/commits/f11d300c16a4c67a2530d827ba9549a8e908f338)

Tests will be fixed in a different commit, cause now they won't run because of the contract size limit errro